### DISCUSSION TITLE
Hi I am having problem in the installation Of Seurat

### DISCUSSION THREAD

**Discussion by alquamalok22** [satijalab/seurat#8250] [General]
Hi I am having problem in the installation Of Seurat? It s giving me error everytime i try to install
library(Seurat)
Loading required package: SeuratObject
Error: package or namespace load failed for ‘SeuratObject’ in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
 namespace ‘Matrix’ 1.6-1.1 is already loaded, but >= 1.6.3 is required
Error: package ‘SeuratObject’ could not be loaded

**Comment by Shangsi-Lin** [comment]
Hi, I encountered a similar issue a while ago, when you are installing the package, there will be a prompt asking something about downloading from a compilation source, you need to click "no" for that prompt and it should be fine. Hope this helps!
